training/src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,102 @@
+package wolox.training.models;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.validation.constraints.NotNull;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+import wolox.training.exceptions.BookNotFoundException;
+
+@Entity(name="users")
+public class User {
+
+    @Id
+    @GeneratedValue
+    private Long id;
+
+    @NotNull
+    private String username;
+
+    @NotNull
+    private String name;
+
+    @NotNull
+    private LocalDate birthdate;
+
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "book_user",
```

Comment:
Join table annotation is not necessary unless you want to re-define the names and columns.
Otherwise, by simply specifiyng the ManyToMany annotation, this generates a jointable by default.